components/aws-classifier-lambda/README.md

```diff
@@ -0,0 +1,54 @@
+
+The lambda in question was built using Typescript and is built and deployed using the [Serverless](https://www.serverless.com/) framework.
+
+# Pre-Requisites
+Before beginning this part of the tutorial please make sure you have done the following steps:
+
+1. Have a valid AWS account that is able to access and deploy AWS API Gateway and AWS Lambda. 
+
+2. Have a set of AWS credentials (eg. client id and secret). For more information on setting up your AWS credentials on your local machine see [here](https://docs.aws.amazon.com/sdkref/latest/guide/creds-config-files.html).
+
+3. Install the Serverless framework on the machine you are going to run this deploy from. The documentation for downloading and installing the Serverless framework can be found [here](https://www.serverless.com/framework/docs/getting-started/).
+
+4. Install NodeJS on your machine if you do not already have it. For this blueprint, I used version 14.15.0. If you do not have NodeJS on your machine, I recommend using [nvm](https://github.com/nvm-sh/nvm) (Node Version Manager) to install Node.
+
+# Deployment Steps
+
+1. **Create a `.env.dev` file in the `components/aws-classifier-lambda/dev` directory.  This file should contain 2 values: `CLASSIFIER_ARN` and `CLASSIFIER_CONFIDENCE_THRESHOLD`.  The `CLASSIFIER_ARN` should be set to `EndpointArn` created when you trained the classifier (reference the `components/aws-comprehend` on how to train the classifier). The `CLASSIFIER_CONFIDENCE_THRESHOLD` is a value between 0 and 1 that signifies the level of confidence you want the lambda to have before returning a classification. For example, if `CLASSIFIER_CONFIDENCE_THRESHOLD` equals .75, that means the classification returned by the AWS Comprehend classifier must be at or above 75% to return the classification. If the classification falls below this value, the lambda will return an empty string for the classification.  Shown below is an example `.env.dev` file.
+
+```
+CLASSIFIER_ARN=arn:aws:comprehend:us-east-1:000000000000:document-classifier-endpoint/emailclassifier-example-only
+CLASSIFIER_CONFIDENCE_THRESHOLD=.75
+```
+
+If you did not write down the `EndpointArn` you can use the AWS cli command: `aws comprehend list-endpoints` command to retrieve the endpoint. The `EndpointArn` will be in the returned data.
+
+2. **Open a command-line window and in the `email-aws-comprehend-blueprint/components/aws-classifier-lambda` directory run the `npm i` command to download and install all of the third-party packages and dependences**.  
+
+3. **Deploy the lambda using `sls deploy` command.** This will take about a minute to deploy and when the lambda is deploy there are two important pieces of information that need to be captured for using when deploying the Genesys Cloud flow:  `api key` and `endpoints`.
```

Comment:
User `serverless deploy` instead of `sls` because it conflicts with a Powershell cmdlet alias.

Reply:
Good call. fixed